src/components/main/navbar/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styles from '../../../css-modules/main.module.css';
+
+// Prop Types
+import PropTypes from 'prop-types';
+
+const NavBar = ({ children, typeSelected, setTypeSelected, sortBySelected, setSortBySelected }) => {
+
+    const changeItemSelected = (type) => {
+        setTypeSelected(type)
+    }
+
+    return (
+        <section>
+            <nav className={ styles.navbar }>
+                <ul className={ styles.navbarItems }>
+                    <li className={ typeSelected === 'all' ? styles.navbarItemActive : styles.navbarItem } onClick={ () => changeItemSelected('all') }>All</li>
```

Comment:
It's better to follow DRY practice. So here you can create an array with film types and iterate through it. Smth like this:
```js
CATEGORIES.map(category => <li key={category} className={ typeSelected === category ? styles.navbarItemActive : styles.navbarItem } onClick={ () => changeItemSelected(category) }>{category}</li>)
```

Reply:
Yeah, I think the same!